examples/dynamic_example.jl

```diff
@@ -146,7 +145,12 @@ function init_problem()
     x0 = X0
     aleas = generate_probability_laws()
 
-    x_bounds = [(VOLUME_MIN, VOLUME_MAX), (VOLUME_MIN, VOLUME_MAX)]
+    #Define bounds for the state and the control
+    x_bounds  = []
+    for u = 1:DIM_STATES
+        x_bounds = push!(x_bounds, (VOLUME_MIN, VOLUME_MAX))
+    end
```

Comment:
You could use list interpretation: 
x_bounds = [(Vmin, Vmax) for i in 1:DIM_STATES]